mne_bids/path.py

```diff
@@ -355,6 +445,52 @@ def _check(self):
                                  f'{ALLOWED_FILENAME_SUFFIX}.')
 
 
+def _get_matching_bidspaths_from_filesystem(bids_path):
+    """Get matching file paths for a BIDS path.
+
+    Assumes suffix and/or extension is not provided.
```

Comment:
sorry I am getting to these late ....

raise an error if it's not provided? why assume silently

Reply:
> sorry I am getting to these late ....

Sorry about that, should we hold off with the merging a little longer in the future?

Reply:
go with the flow as long as code has been reviewed by one or two maintainers :) I'll catch up when I can.